### ISSUE TITLE
Getting zero count for every request

### ISSUE THREAD

**Issue by tarzzz** [PyGithub/PyGithub#319]
Hi
I am trying to use the API to get the number of repositories for a particular `created` interval, but I am getting zero results.

Here is the code snippet I am trying to execute:

```
In [1]: import github as gt   
In [8]: g = gt.Github("my_token")
In [9]: f = g.search_repositories("location:California created:2008-01-01..2008-01-28")
In [10]: f.totalCount
Out[10]: 0
```

I have tried changing the location as well as created interval, but still getting zero results.
The same search query gives results in github search bar, though:

![image](https://cloud.githubusercontent.com/assets/1546532/8143746/35af5f8a-11d9-11e5-9b34-5b9792b86547.png)

I have also tried using `search_issues` with similar filters, but getting zero count for the same as well.
`search_users` seem to work fine though.

**Comment by era** [comment]
Hello @tarzzz, 

The problem seems to be the "location:California". If you hit [the api at](https://api.github.com/search/repositories?sort=stars&q=created%3A2008-01-01..2008-01-28&order=desc) using only the created:2008-01-01..2008-01-28 you will receive the 39 results (this is consistent with the PyGithub). If you include the ["location:California"](https://api.github.com/search/repositories?sort=stars&q=location%3ACalifornia+created%3A2008-01-01..2008-01-28&order=desc) you will receive 0 results.

I tried to find in the [Github documentation](https://developer.github.com/v3/search/#search-repositories) the keyword location and I didn't find. If you saw that on the documentation, could you please provide the link?

To the maintainers, if this is not a bug, should we remove any search qualifiers that is not supported by github before every request to avoid this type of problem?

**Comment by michaelrupert-wf** [comment]
I believe that for some reason, search issues doesnt seem to work with multiple queries, I"m trying to find issues with mentions:user+state:open, and getting the same error

**Comment by sfdye** [comment]
This should be fixed in #820
